concordium-sdk/src/main/java/com/concordium/sdk/crypto/CryptoJniNative.java

```diff
@@ -0,0 +1,17 @@
+package com.concordium.sdk.crypto;
+
+public class CryptoJniNative {
```

Comment:
Documentation on the functions here. In particular the return values.

Reply:
done